drivers/i2c/i2c_ll_stm32_v1.c

```diff
@@ -133,13 +298,24 @@ static inline void handle_rxne(I2C_TypeDef *i2c, struct i2c_stm32_data *data)
 			*data->current.buf = LL_I2C_ReceiveData8(i2c);
 			data->current.buf++;
 		}
+	} else {
+
+		if (data->current.flags & I2C_MSG_STOP) {
+			LL_I2C_GenerateStopCondition(i2c);
+		}
+		LL_I2C_TransmitData8(i2c, 0);
```

Comment:
Is it needed?

Reply:
oups. Was used as crutch in test. Thanks, removed.